.github/workflows/ci.yml

```
@@ -180,6 +207,41 @@ jobs:
           path: _build
           retention-days: 14
 
+  generate-full-simulation-report:
+    name: Generate the final report
```

```suggestion
    name: Generate clock control simulation report
```

It isn't obvious what this step is from the UI without mentioning clock control specifically.